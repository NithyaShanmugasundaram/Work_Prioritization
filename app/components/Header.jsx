"use client"
import Container from 'react-bootstrap/Container';
import Nav from 'react-bootstrap/Nav';
import Navbar from 'react-bootstrap/Navbar';
import Link from 'next/link';
import styles from "./../styles/Common.module.css";
function Header() {
  return (
    <Navbar expand="lg" sticky="top" className={`${styles.wrapperContainer}`}>
      <Container fluid>
        <Navbar.Brand href="/" ><div className={styles.navbrand}>Work Prioritization</div></Navbar.Brand>
        <Navbar.Toggle aria-controls="basic-navbar-nav" id={styles.toggle_icon}/>
        <Navbar.Collapse id="basic-navbar-nav" className="justify-content-end">
          <Nav>
            <Link className={styles.navlink} href="/">User Stories</Link>
            <Link className={styles.navlink} href="/tasks">Tasks</Link>
            <Link className={styles.navlink} href="/resource">Resource</Link>
          </Nav>
        </Navbar.Collapse>
      </Container>
    </Navbar>
  );
}

export default Header;