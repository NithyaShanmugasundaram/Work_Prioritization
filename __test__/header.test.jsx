import React from 'react';
import renderer from 'react-test-renderer';
import Header from './../app/components/Header';

test('MyComponent snapshot', () => {
    const component = renderer.create(<Header />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });