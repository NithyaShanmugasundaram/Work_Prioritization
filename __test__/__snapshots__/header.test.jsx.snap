// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyComponent snapshot 1`] = `
<nav
  className="wrapperContainer navbar navbar-expand-lg navbar-light sticky-top"
>
  <div
    className="container-fluid"
  >
    <a
      className="navbar-brand"
      href="/"
    >
      <div
        className="navbrand"
      >
        Work Prioritization
      </div>
    </a>
    <button
      aria-controls="basic-navbar-nav"
      aria-label="Toggle navigation"
      className="navbar-toggler collapsed"
      id="toggle_icon"
      onClick={[Function]}
      type="button"
    >
      <span
        className="navbar-toggler-icon"
      />
    </button>
    <div
      aria-expanded={null}
      className="justify-content-end navbar-collapse collapse"
      id="basic-navbar-nav"
    >
      <div
        className="navbar-nav"
        onKeyDown={[Function]}
      >
        <a
          className="navlink"
          href="/"
          onClick={[Function]}
          onMouseEnter={[Function]}
          onTouchStart={[Function]}
        >
          User Stories
        </a>
        <a
          className="navlink"
          href="/tasks"
          onClick={[Function]}
          onMouseEnter={[Function]}
          onTouchStart={[Function]}
        >
          Tasks
        </a>
        <a
          className="navlink"
          href="/resource"
          onClick={[Function]}
          onMouseEnter={[Function]}
          onTouchStart={[Function]}
        >
          Resource
        </a>
      </div>
    </div>
  </div>
</nav>
`;
